src/java.base/share/classes/java/lang/ref/package-info.java

```diff
@@ -152,8 +150,7 @@
  *
  * <li> The enqueueing of a reference (by the garbage collector, or
  * by a successful call to {@link Reference#enqueue}) <i>happens-before</i>
- * the reference is removed from the queue ("dequeued") by {@link ReferenceQueue#poll} or
- * {@link ReferenceQueue#remove}.</li>
+ * the reference is removed from the queue (<em>dequeued</em>).</li>
```

Comment:
@bchristi-git I wonder if it makes sense to use the same style when referring to "dequeue", so either always within quotes or only using <em></em> (see line 108)

Reply:
Sure; I'll remove the quotes.